### DISCUSSION TITLE
Release v1.8.4 / 5.63.4

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#2801] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This build fixed a compatibility issue with MSEdge 112.x as well as a few other bugs.

Please review the full [Changelog](https://github.com/sandboxie-plus/Sandboxie/blob/master/CHANGELOG.md#184--5634---2023-04-07) for more details.

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.8.4'>Release v1.8.4 / 5.63.4</a>.</em>

**Comment by ghost** [comment]
Does Edge 111.0.1661.54 crash in orange box or it is just me?

**Reply by offhub** [reply]
No crash with:
```
Microsoft Edge	111.0.1661.62 (Official build) (64-bit) 
Operating system	Windows 11 Version 22H2 (Build 22621.1413)
```
```

Microsoft Edge	111.0.1661.51 (Official build) (64-bit) 
Operating system	Windows 10 Version 21H2 (Build 19044.2728)
```

**Reply by ghost** [reply]
Thank you. It turned out I have an issue with XMP
